src/app/core/models/issue.model.ts

```diff
@@ -107,6 +114,6 @@ export const phaseTesterResponseDescriptionTemplate = new RegExp('(?<header># De
   '|## State the duplicated issue here, if any|# Items for the Tester to Verify|$)', 'gi');
 
 export const phaseModerationDescriptionTemplate = new RegExp('(?<header># Description|# Team\'s Response|## State the duplicated issue ' +
-  'here, if any|## Proposed Assignees|# Items for the Tester to Verify|# Tutor\'s Response|## Tutor to check)\\s+' +
-  '(?<description>[\\s\\S]*?)(?=# Team\'s Response|## State the duplicated issue here, if any|## Proposed Assignees|' +
-  '# Items for the Tester to Verify|# Tutor\'s Response|## Tutor to check|$)', 'gi');
+  'here, if any|# Disputes|# Tutor\'s Response|## Tutor to check)\\s+' +
```

Comment:
@CATcher-org/admins 

A query, should we make the regex strict or allow some flexibility? For example in this regex, it checks for exactly the phrase `... Tutor's Response`, should we tweak it such that it accepts `... tutor's response` as well. Or for example when we are checking for a paragraph we often mark it with a preset number of new lines that is expected to exist between lines, should we make it such that the number of new lines are variable? It will allow the app to function even if there are minor changes in the formatting (in the case that it occurs) but I'm not sure if its necessary. Please advice.